syntax = "proto3";

package fbsrankings.messages.query;

import "fbsrankings/messages/options/options.proto";

message SeasonResult {
    string season_id = 1;
    int32 year = 2;
}

message SeasonsResult {
    string query_id = 1;
    repeated SeasonResult seasons = 2;
}

message SeasonsQuery {
    string query_id = 1;

    option (fbsrankings.messages.options.topic) = "fbsrankings.query.seasons";
}
